### PR TITLE
tx model hub update

### PR DESCRIPTION
**Pull request by yjh321**

### CODE REVIEW

**Comment by dustjr1565** [comment]
Please implement test code in the 'tests' folder

**Review by siegkeron — APPROVED** [review]

**Review by dustjr1565 — APPROVED** [review]

**Comment by ZeroAct** [comment]
tx_model cannot be installed with pip. So, it cannot be tested on github action.

Push test code and just make it sure to work well.

**Review by ZeroAct — CHANGES_REQUESTED** [review]
tx_model cannot be installed with pip. So, it cannot be tested on github action.

Push test code and just make it sure to work well.

File: waffle_hub/hub/adapter/tx_model/tx_model_hub.py [hunk 1]
```diff
@@ -128,13 +134,41 @@ def get_preprocess(self, task: str, *args, **kwargs):
...
+            def preprocess(x, *args, **kwargs):
+                return normalize(x)
+
         return preprocess
 
     def get_postprocess(self, task: str, *args, **kwargs):
 
         if task == "object_detection":
 
+            def inner(
```
Comment (siegkeron) [inline waffle_hub/hub/adapter/tx_model/tx_model_hub.py:149]: It seems copy postprocess from ultralytics hub. Is it correct?

**Review by siegkeron — CHANGES_REQUESTED** [review]

File: waffle_hub/hub/adapter/tx_model/tx_model_hub.py (hunk 1 above)
Comment (yjh321) [reply waffle_hub/hub/adapter/tx_model/tx_model_hub.py]: As you said, the code before the change was supposed to divide the bbox by the height and width of the image like Ultralytics. However, I noticed that the output of tx model detector contains bbox[x, y, w, h] in the range of 0~1, confidence score, and class ids, so I changed it to no longer divide the bbox by the height and width of the image.

File: waffle_hub/hub/adapter/tx_model/configs/model_cfg.py [hunk 2]
```diff
@@ -6,58 +6,95 @@ def get_model_config(
...
+                            "fl_alpha": -1,
+                            "fl_gamma": 0.0,
+                            "label_smoothing": 0.0,
+                        }
+                    },
+                }
+            },
+            "optim": {
+                "name": "SGD",
+                "lr": 0.001,
```
Comment (ZeroAct) [inline waffle_hub/hub/adapter/tx_model/configs/model_cfg.py:45]: is `lr` fixed?
use `learning_rate` parameter of `TrainConfig`

Comment (yjh321) [reply waffle_hub/hub/adapter/tx_model/configs/model_cfg.py]: It has been fixed

**Review by ZeroAct — APPROVED** [review]

**Pull request merged**